### ISSUE TITLE
Website a11y: Some front page images are missing alt text

### ISSUE THREAD

**Issue by julianna-ciq** [finos/FDC3#942]
## Minor Issue
Please use minor issues for discrepancies or errors found in the spec or supporting documentation, examples, and other materials.  If you wish to propose a new feature or a revision to an existing feature, please use the *Enhancement Request* template.

### Area of Issue
[ ] App Directory
[ ] API
[ ] Context Data
[ ] Intents
[ ] Use Cases
[x] Other

### Issue Description: 

On the FDC3 homepage, the following images don't have alt text:
1. the FDC3 logo (Recommended alt text: "FDC3")
2. the section images for Intents/Context Data/App Directory (Recommended alt text: none)
3. the conformance badges (Recommended alt text: "FDC3 1.2 Certified", "FDC3 2.0 Certified")

This means that a screen reader user will hear something like "fdc3-logo-2019.png" instead of "FDC3".

I am willing to make this change.

**Comment by julianna-ciq** [comment]
https://github.com/finos/FDC3/pull/950

**Comment by kriswest** [comment]
resolved by #950
